tests/acceptance/features/apiGraph/getGroup.feature

```diff
@@ -52,3 +52,54 @@ Feature: get groups and their members
     When user "Brian" gets all the members of group "tea-lover" using the Graph API
     Then the HTTP status code should be "401"
     And the last response should be an unauthorized response
+
+
+    Scenario: admin user gets all groups along with its member's information
+      Given user "Brian" has been created with default attributes and without skeleton files
+      And user "Carol" has been created with default attributes and without skeleton files
+      And group "tea-lover" has been created
+      And group "coffee-lover" has been created
+      And user "Alice" has been added to group "tea-lover"
+      And user "Brian" has been added to group "coffee-lover"
+      And user "Carol" has been added to group "tea-lover"
+      When user "Alice" retrieves all groups along with their members using Graph API
+      Then the HTTP status code should be "200"
+      And the group 'coffee-lover' should have the following member information
+        | displayName  | id        | mail              | onPremisesSamAccountName |
+        | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    |
+      And the group 'tea-lover' should have the following member information
+        | displayName  | id        | mail              | onPremisesSamAccountName |
+        | Alice Hansen | %uuid_v4% | alice@example.org | Alice                    |
+        | Carol King   | %uuid_v4% | carol@example.org | Carol                    |
+
+
+  Scenario: normal user gets all groups along with their members information
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And group "tea-lover" has been created
+    And group "coffee-lover" has been created
+    And user "Alice" has been added to group "tea-lover"
+    And user "Brian" has been added to group "coffee-lover"
+    When user "Brian" retrieves all groups along with their members using Graph API
+    Then the HTTP status code should be "401"
+    And the last response should be an unauthorized response
```

Comment:
`401`

Reply:
`And the last response should be an unauthorized response` this steps basically checks the message from the response. And all others scenarios have this step after `And the last response should be an unauthorized response`.